### DISCUSSION TITLE
[Flows] Groups flows

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#15180] [Feature Requests]
While making some flows i noticed the following problem: you lose sight over all flows really quick (for example every flow attached to an lead).

Therefor i think its the best to make one of the following 2 things:
1. A tag system with an easy option to filter on it.
You can basicly create/add tags to a flow and it will appear in the overview, if you click on it, it will be added as a filter for the GET request.
2. A folder system
Basicly the same as how you setup files/folders.

**Comment by azrikahar** [comment]
Perhaps a more direct approach (and may be more consistent with other parts of the app) is to update it to use layout-wrapper, similar to how `Settings -> Presets & Bookmarks` and `Settings -> Webhooks` has the search bar and filters.

Definitely a good idea as an improvement to Flows, along with other great suggestions in #15134.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
